Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -172,7 +187,7 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
           </Text>
         </Flex.Item>
         <Flex.Item size="size.quarter" variables={{ 'size.quarter': '24%' }}>
-          {this.sendIndicator(message)}
+          {this.sendingNumbers(message)}
```

Comment:
Does this.sendIndicator() need to be deleted?

Reply:
deleted